### DISCUSSION TITLE
Problems Needed to be addressed in upcoming Docker Blog

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1826] [Ideas]
I am about to write a Blog on Docker which would contain everything one needs to know to get started & some resources as well !!
So, wanted suggestions from folks who are interested in learning about Docker that what do they find difficult or what should i must include in that, to make it effective & be able to help new folks to get started !!

Fill in your suggestions in the thread below 😇

**Comment by verma-kunal** [comment]
P1 - I understand the basic concept. But when they show the docker files they are creating and commands they are running then I am not able to visualize it and end up not understanding how to use them or write them from scratch
(@kaiwalyakoparkar )

**Reply by kaiwalyakoparkar** [reply]
Thank you for adding (Sorry I was late to add)

**Reply by verma-kunal** [reply]
No worries at all 🙂

**Comment by verma-kunal** [comment]
P2 - I want to know why should I dockerize my project when I am not sharing them most of the time. (@ashwinexe )

**Reply by ashwinexe** [reply]
P3 - How hard is docker (first thing a beginner tends to avoid is non-graphical scary looking code and commands)

**Comment by schmelto** [comment]
You have to send me the link when you have published the blog I'm not into Docker that much but I want to know more about it 🚀

**Reply by verma-kunal** [reply]
Sure! I'll send that in the community !! Would take time to write, but hope that would help 🙂
